sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -118,42 +123,42 @@ internal static RemoteDependency ConvertToDependencyDocument(Activity activity)
             switch (liveMetricsTagsProcessor.ActivityType)
             {
                 case OperationType.Http:
-                    remoteDependencyDocument.Name = activity.DisplayName;
-                    remoteDependencyDocument.CommandName = AzMonList.GetTagValue(ref liveMetricsTagsProcessor.Tags, SemanticConventions.AttributeUrlFull)?.ToString();
+
+                    var httpUrl = AzMonList.GetTagValue(ref liveMetricsTagsProcessor.Tags, SemanticConventions.AttributeUrlFull)?.ToString();
+                    remoteDependencyDocument.Name = liveMetricsTagsProcessor.Tags.GetNewSchemaHttpDependencyName(httpUrl) ?? activity.DisplayName;
```

Comment:
What is the issue with previous functionality? Please note live metrics sends data every second, minimal processing of attributes are the best. I recommend something like below.

AzMonList.GetTagValue(ref liveMetricsTagsProcessor.Tags, SemanticConventions.AttributeUrlFull)?.ToString() ?? activity.DisplayName

Reply:
The previous implementation set `remoteDependencyDocument.Name = activity.DisplayName;` and all other Dependency types do this.

But the goal with this PR was to match the mappings being made in the Exporter.
https://github.com/Azure/azure-sdk-for-net/blob/125bfbb68c585771c6c60f32e0542d45bfc20cfe/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RemoteDependencyData.cs#L71-L72

Reply:
I recommend to keep this simple, considering its not stored anywhere.